src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OperatorOverloadsHaveNamedAlternates.Fixer.cs

```diff
@@ -26,11 +28,123 @@ public sealed override FixAllProvider GetFixAllProvider()
+                            accessibility: Accessibility.Public,
+                            modifiers: expectedSignature.IsStatic ? DeclarationModifiers.Static : DeclarationModifiers.None,
+                            statements: bodyStatements);
+                    }
+
+                    var newTypeDeclaration = generator.AddMembers(typeDeclaration, addedMember);
+                    return context.Document.WithSyntaxRoot(root.ReplaceNode(typeDeclaration, newTypeDeclaration));
+                case OperatorOverloadsHaveNamedAlternatesAnalyzer.FixVisibilityText:
+                    var badVisibilityNode = generator.GetDeclaration(node, DeclarationKind.Method) ?? generator.GetDeclaration(node, DeclarationKind.Property);
+                    var badVisibilitySymbol = semanticModel.GetDeclaredSymbol(badVisibilityNode, context.CancellationToken);
+                    var symbolEditor = SymbolEditor.Create(context.Document);
+                    var newSymbol = await symbolEditor.EditOneDeclarationAsync(badVisibilitySymbol,
+                        (documentEditor, syntaxNode, ct) => Task.Run(() => documentEditor.SetAccessibility(badVisibilityNode, Accessibility.Public))).ConfigureAwait(false);
+                    var newDocument = symbolEditor.GetChangedDocuments().Single();
+                    var newRoot = await newDocument.GetSyntaxRootAsync(context.CancellationToken);
+                    return context.Document.WithSyntaxRoot(newRoot);
+                default:
+                    return context.Document;
+            }
+        }
+
+        private static ExpectedMethodSignature GetExpectedMethodSignature(IMethodSymbol operatorOverloadSymbol, Compilation compilation)
+        {
+            var containingType = (ITypeSymbol)operatorOverloadSymbol.ContainingType;
+            var returnType = operatorOverloadSymbol.ReturnType;
+            var expectedName = OperatorOverloadsHaveNamedAlternatesAnalyzer.GetExpectedAlternateMethodGroup(operatorOverloadSymbol.Name, returnType).AlternateMethod1;
+            switch (operatorOverloadSymbol.Name)
+            {
+                case "op_GreaterThan":
+                case "op_GreaterThanOrEqual":
+                case "op_LessThan":
+                case "op_LessThanOrEqual":
+                    // e.g., public int CompareTo(MyClass other)
+                    var intType = compilation.GetSpecialType(SpecialType.System_Int32);
+                    return new ExpectedMethodSignature(expectedName, intType, ImmutableArray.Create(Tuple.Create("other", containingType)), isStatic: false);
+                case "op_Decrement":
+                case "op_Increment":
+                case "op_UnaryNegation":
+                case "op_UnaryPlus":
+                    // e.g., public MyClass Decrement(MyClass item)
+                    return new ExpectedMethodSignature(expectedName, returnType, ImmutableArray.Create(Tuple.Create("item", containingType)));
```

Comment:
`isStatic` is `true` by default. Is the comment wrong? That is, should the example be `public static MyClass Decrement(MyClass item)`?